src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -152,12 +194,13 @@ const AppWrapper = (props: { children: ReactNode }) => {
 
     try {
       await AriesAgent.agent.start();
+      setIsOffline(false);
     } catch (e) {
       // @TODO - foconnor: Should specifically catch the error instead of all, but OK for now.
-      setAgentInitErr(true);
       // eslint-disable-next-line no-console
-      console.error(e);
-      return;
+      AriesAgent.agent.bootAndConnect().then(() => {
```

Comment:
Can we check here that the error is an issue with connecting to KERIA? And re-throw if it's some other issue...